lib/skin.js

```diff
@@ -15,10 +15,14 @@ module.exports = function (gulp, gutil) {
   var skinPropertyCopySass = 'copy_sass_files_to_child';
   var baseFeaturePath = gutil.env.newStructure ? 'plugin/res/feature/responsivebase/' : 'res/feature/responsivebase/';
   var peakFeaturePath = gutil.env.newStructure ? 'plugin/res/feature/responsivepeak/' : 'res/feature/responsivepeak/';
+  var themeFeaturePath = 'other/res/skins'
   var skinsBaseDir = 'res/skins';
   var communityPluginsDir = 'other';
   var propFileName = 'skin.properties';
-
+  var checkTheme, version;
+  var supportedMinVersionForThemeCheck = '22.7';
```

Comment:
Hope this is still true given that community is now in 22.8 version in active branch.

Reply:
yes, this will still hold true as the lia changes for this fix already went in 22.7 fix version